Properties/Resources.resx

```diff
@@ -0,0 +1,247 @@
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="MustNotBeNullOrEmpty_string" xml:space="preserve">
+    <value>{0} must not be null or empty.</value>
+    <comment>{0} - The name of the parameter</comment>
+  </data>
+  <data name="InvalidEntryType_string_type" xml:space="preserve">
+    <value>{0} is of an invalid entry type, expected {1} but got {2}.</value>
+    <comment>{0} - The name of the parameter.
+{1} - The expected type of the parameter.
+{2} - The actual type of the parameter.</comment>
+  </data>
+  <data name="UnsupportedPermission_UnixExternalPermissions" xml:space="preserve">
+    <value>Unsupported Permission {0}. Files other than regular ones, directories and symlinks aren't supported yet.</value>
+    <comment>{0} - The name of the permission.</comment>
+  </data>
+  <data name="MustBeAnInstanceOf_string_type" xml:space="preserve">
+    <value>{0} must be an instance of {1}.</value>
+    <comment>{0} - The name of the parameter.
+{1} - The type the parameter should be.</comment>
+  </data>
+  <data name="ExpectedAnInstanceOf_string_type" xml:space="preserve">
+    <value>Expected an instance of {0} for {1} but got {2}</value>
+    <comment>{0} - The expected type of the parameter.
+{1} - The name of the parameter.
+{2} - The actual type of the parameter.</comment>
+  </data>
+  <data name="UnableToDetermineFileType_file" xml:space="preserve">
+    <value>Failed to determine type of the '{0}' file.</value>
+    <comment>{0} - A filename</comment>
+  </data>
+  <data name="FailedToReadSymLink_link_error" xml:space="preserve">
+    <value>Could not read the symbolic link '{0}': {1}.</value>
+    <comment>{0} - The symbolic link that was being read.
+{1} - The actual error message reported by the operating system.</comment>
+  </data>
+  <data name="FileTypeNotSupported_filetype" xml:space="preserve">
+    <value>Storing files of type {0} is not supported.</value>
+    <comment>{0} - The type of the file.</comment>
+  </data>
+  <data name="FailedToSetOwner_entry_error_oserror" xml:space="preserve">
+    <value>Warning: failed to set owner of entry '{0}' ({1}:{2}).</value>
+    <comment>{0} - A filename.
+{1} - The error message.
+{2} - The operating system error.</comment>
+  </data>
+  <data name="FailedToStatFile_file_error" xml:space="preserve">
+    <value>Warning: failed to stat file '{0}': {1}.</value>
+    <comment>{0} - A filename.
+{1} - The error message.</comment>
+  </data>
+  <data name="UnexpectedOptionsType_string_type_type" xml:space="preserve">
+    <value>Unexpected options type for {0}. Expected {1} but got {2}.</value>
+    <comment>{0} - The parameter name.
+{1} - The expected type of the parameter.
+{2} - The actual type of the parameter.</comment>
+  </data>
+  <data name="FileAlreadyExists" xml:space="preserve">
+    <value>The file already exists.</value>
+    <comment></comment>
```

Comment:
There's a little gotcha with the XliffTasks tooling that it doesn't like empty `<comment></comment>` elements (<https://github.com/dotnet/xliff-tasks/issues/342>), so all of the empty elements can be removed to make XliffTasks happy.